cabal.project

```diff
@@ -199,3 +200,20 @@ source-repository-package
       libs/small-steps
       libs/small-steps-test
       libs/non-integral
+
+-- Includes cardano-testnet fix for Babbage era on fork of 1.35.4
+-- PR for master branch: https://github.com/input-output-hk/cardano-node/pull/4729
+source-repository-package
+    type: git
+    location: https://github.com/james-iohk/cardano-node
+    tag: ae809bb8d216b24341da82a8d441f192cebd9aaa
+    subdir:
+      cardano-testnet
+
+-- QueryConvenienceError deriving Show
```

Comment:
This is a minor change to cardano-api for `QueryConvenienceError` to derive Show to allow out tests to error via GHC callStack. It doesn't currently have this because errors are produced as Text for cardano-cli to display.